defs/scenarios/system/bugs.yaml

```diff
@@ -0,0 +1,26 @@
+checks:
+  has_1971565:
+    requires:
+      apt:
+        mysql-router:
+          # focal:
+          - min: 8.0.29-0ubuntu0.20.04.2
+            max: 8.0.29-0ubuntu0.20.04.2
+          # impish:
+          - min: 8.0.29-0ubuntu0.21.10.1
+            max: 8.0.29-0ubuntu0.21.10.1
+          # jammy:
+          - min: 8.0.29-0ubuntu0.22.04.1
+            max: 8.0.29-0ubuntu0.22.04.1
+conclusions:
+  lp1971565:
+    decision: has_1971565
+    raises:
+      type: LaunchpadBug
+      bug-id: 1971565
+      message: >-
+        Installed package '{package}' with version {version} has a known bug
+        that prevents mysql to start. To fix this you need to the latest version.
```

Comment:
Also, I get that this is going into the system plugin because there is no proper home for it and that reminds me that have a for a while now intended on having a mysql plugin e.g. for https://github.com/canonical/hotsos/issues/76. So given this patch I think I'll use that as an excuse to create one and then we can move this under the mysql plugin.

Reply:
Ive submitted a new mysql plugin - https://github.com/canonical/hotsos/pull/345 - if we can get that landed I think we should move your scenario to defs/scenarios/mysql/